li-utils/src/main/javaPegasus/com/linkedin/common/urn/DatasetFieldUrn.java

```diff
@@ -55,22 +51,47 @@ public DatasetFieldUrn coerceOutput(Object object) throws TemplateOutputCastExce
     }, DatasetFieldUrn.class);
   }
 
+  public DatasetFieldUrn(String dataPlatform, String datasetName, FabricType fabricType, String fieldPath) {
+    this(new DatasetUrn(new DataPlatformUrn(dataPlatform), datasetName, fabricType), fieldPath);
+  }
+
   /**
-   * Creates a new instance of a {@link DatasetFieldUrn }.
+   * Creates a new instance of a {@link DatasetFieldUrn}.
    *
-   * @param dataset Dataset that this dataset field belongs to.
-   * @param fieldPath Dataset field path or column name
+   * @param datasetUrn dataset that this dataset field belongs to
```

Comment:
nit: param is `dataset`

Reply:
nvm, you changed input variable name too

Reply:
I did rename the param, but yeah, lets be consistent with the PDL. Thanks for pointing that out.